docs/index.md

```diff
@@ -10,10 +10,12 @@ PROJECT_FLOW
 VSCODE_DEVELOPMENT
 api/index
 ci-cd/index
+development_controllers/index
 discussion/index
 getting_started/index
 cluster_and_device_type_dev/index
```

Comment:
Does it make more sense to have the (zap|spec)_clusters files in here?  Or maybe more towards the top rather than at the bottom?

I feel like that's a page a lot of people will want to visit.

Reply:
Possibly, but they need a re-vamp anyway. That's a bit orthogonal to this change through. Can do in a follow up.

Reply:
discussed offline - follow up ok.